lcli/src/main.rs

```diff
@@ -552,6 +553,58 @@ fn main() {
                         .display_order(0)
                 )
         )
+        .subcommand(
+            Command::new("rescue")
```

Comment:
Worth adding a note that this tool only works if the distance is within the data availability windows (4096 epochs ~ 18 days) otherwise blobs will become unavailable.